### PR TITLE
Add device names to sms alerts

### PR DESCRIPTION
**Pull request by dabulashvili-zz**

### CODE REVIEW

**Comment by DatoBula** [comment]
[More Descriptive SMS Alerts](https://trello.com/c/XLTxpiPl/742-more-descriptive-sms-alerts)

File: lib/notifier.js [hunk 1]
```diff
@@ -234,7 +234,9 @@ function alertSubject (alertRec) {
   if (alerts.length === 0) return null
 
   const alertTypes = _.map(codeDisplay, _.uniq(_.map('code', alerts))).sort()
-  return '[Lamassu] Errors reported: ' + alertTypes.join(', ')
+  const deviceNames = _.values(alertRec.deviceNames)
+
+  return '[Lamassu] Errors reported: ' + alertTypes.join(', ') + ' (' + deviceNames.join(', ') + ')'
```
Comment (joshmh) [inline lib/notifier.js]: This isn't quite according to the Trello card spec. The idea was to have each alertType individually show which machines it's affecting, not to list all machines affected by all alerts in the end.

**Review by joshmh — CHANGES_REQUESTED** [review]

File: lib/notifier.js [hunk 2]
```diff
@@ -226,15 +228,24 @@ function printEmailAlerts (alertRec) {
...
-
   _.keys(alertRec.devices).forEach(function (device) {
     alerts = _.concat(alerts, alertRec.devices[device])
   })
 
   if (alerts.length === 0) return null
 
-  const alertTypes = _.map(codeDisplay, _.uniq(_.map('code', alerts))).sort()
-  return '[Lamassu] Errors reported: ' + alertTypes.join(', ')
+  const alertTypes = _.map(function (el) {
```
Comment (joshmh) [inline lib/notifier.js]: Use arrow notation.

File: lib/notifier.js [hunk 3]
```diff
@@ -226,15 +228,24 @@ function printEmailAlerts (alertRec) {
...
+    return {
+      codeDisplay: codeDisplay(el.code),
+      machineName: el.machineName
+    }
+  }, alerts).sort()
+
+  const displayAlertTypes = _.uniq(_.map(function (alertType) {
+    return alertType.codeDisplay + ' (' + alertType.machineName + ') '
```
Comment (joshmh) [inline lib/notifier.js]: Some alerts are global and don't have machineNames. What happens in that case?

**Review by joshmh — CHANGES_REQUESTED** [review]

**Review by joshmh — CHANGES_REQUESTED** [review]

**Pull request merged**